force-app/main/default/classes/AccountIntegrationController.cls

```diff
@@ -1,47 +1,65 @@
 public class AccountIntegrationController {
+    public static String ACCOUNTS_ENDPOINT = '/services/apexrest/Account';
+
     @AuraEnabled
-    // @future(callout=true)
     public static List<Account> getDetails() {
-
-        HttpRequest req = new HttpRequest();
-        req.setEndpoint('callout:ReturnContact/services/apexrest/Account');
-        req.setHeader('content-type', 'aplication/json; charset=UTF-8');
-        req.setHeader('Accept', 'application/json');
-        req.setMethod('GET');
-        Http http = new Http();
-        HttpResponse respon =  http.send(req);
-        
-        System.debug('response---Code------' + respon.getStatusCode());
-        System.debug('response---body------' + respon.getBody());
+        String method = 'GET';
+        String body = '';
         List<Account> accounts = new List<Account>();
 
-        if(respon.getStatusCode() == 200) {
-            List<Object> jsonResults = (List<Object>) JSON.DeserializeUntyped(respon.getBody());
-            System.debug('jsonResults'+jsonResults);
-             for (object ac : jsonResults) {
-                Map<String, Object> acMap = (Map<String, Object>)ac;
+        Map<Integer, String> response = AccountCallouts.executeHttpRequest(ACCOUNTS_ENDPOINT, method, body, false);
+         if(response.containsKey(200)) {
```

Comment:
if you always have only one key in map then you don't have a map. use some object instead (some wrapper for example)